src/main/java/de/tum/in/www1/artemis/web/rest/ExamResource.java

```diff
@@ -683,6 +688,32 @@ public ExamResource(UserRepository userRepository, CourseRepository courseReposi
         return ResponseEntity.ok().body(null);
     }
 
+    /**
+     * DELETE /courses/{courseId}/exams/{examId}/allstudents :
+     * Remove all students of the exam so that they cannot access the exam any more.
+     * Optionally, also deletes participations and submissions of all students in their student exams.
+     *
+     * @param courseId     the id of the course
+     * @param examId       the id of the exam
+     * @param withParticipationsAndSubmission request param deciding whether participations and submissions should also be deleted
+     *
+     * @return empty ResponseEntity with status 200 (OK) or with status 404 (Not Found)
+     */
+    @DeleteMapping(value = "/courses/{courseId}/exams/{examId}/allstudents")
```

Comment:
In terms of REST, we would want to delete the Collection:
```suggestion
    @DeleteMapping(value = "/courses/{courseId}/exams/{examId}/students")
```

Reply:
Thanks for the feedback, done 👍